metricbeat/module/kubernetes/state_cronjob/_meta/fields.yml

```diff
@@ -0,0 +1,44 @@
+- name: cronjob
+  type: group
+  description: >
+    kubernetes cronjob metrics
+  release: beta
+  fields:
+    - name: name
+      type: keyword
+      description: Cronjob name
+    - name: namespace
+      type: keyword
+      description: Cronjob namespace
+    - name: schedule
+      type: keyword
+      description: Cronjob schedule
+    - name: concurrency
+      type: keyword
+      description: Concurrency policy
+    - name: active.count
+      type: long
+      description: Number of active pods for the cronjob
+    - name: is_suspended
+      type: boolean
+      description: Whether the cronjob is suspended
+    - name: created.sec
+      type: double
+      description: Epoch seconds since the cronjob was created
+    - name: lastschedule.sec
```

Comment:
Maybe `last_schedule.sec` or even `schedule.last.sec` so next is also `schedule.next.sec`